server/services/viper.js

```diff
@@ -38,10 +38,6 @@ function viperRatingFromApi(nomisId) {
   });
 }
 
-function ratingFor(db, nomisId) {
-  return viperRatingFromApi(nomisId);
-}
-
-module.exports = function createViperService(db) {
-  return { rating: nomisId => ratingFor(db, nomisId) };
+module.exports = function createViperService() {
+  return { rating: viperRatingFromApi };
```

Comment:
Nice but not part of this ticket?

Reply:
No but I had to do stuff around viperScores since we now require test users